onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -662,56 +603,88 @@ def write_map_to_file(result, file_name):
         file.write(json.dumps(existed_result)) # use `json.loads` to do the reverse
 
 
-def get_system_info():
-    info = {}
+def get_cuda_version():
+    nvidia_strings = get_output(["nvidia-smi"]) 
+    version = re.search(r'CUDA Version: \d\d\.\d', nvidia_strings).group(0) 
+    return version
+    
+def get_trt_version():
+    nvidia_strings = get_output(["dpkg", "-l"])
+    version = re.search(r'nvinfer.*\d\.\d\.\d\-\d', nvidia_strings).group(0)
+    return version
+ 
+# not use for this script temporarily
+def tmp_get_trt_version():
```

Comment:
didn't change/ not in use for this script.

Reply:
is tmp_get_trt_version() not used anymore? should we get rid of it?

Reply:
Definitely not in this script @chilo-ms can I delete?